sycl/include/sycl/detail/pi.h

```diff
@@ -152,6 +152,8 @@
 // 15.43 Changed the signature of piextMemGetNativeHandle to also take a
 // pi_device
 // 15.44 Add coarse-grain memory advice flag for HIP.
+// 15.45 Added piextKernelSuggestMaxCooperativeGroupCount and
+//       piextEnqueueCooperativeKernelLaunch.
 
 #define _PI_H_VERSION_MAJOR 15
 #define _PI_H_VERSION_MINOR 44
```

Comment:
```suggestion
#define _PI_H_VERSION_MINOR 45
```
Not sure if this versioning is important, but since you added the comment above then this version should be changed too.